run_conformance_tests.py

```diff
@@ -90,15 +95,24 @@ def handle_args(argv):
 
     host_names = (args.host_platform_name, args.host_device_name)
     opencl_names = (args.opencl_platform_name, args.opencl_device_name)
+    full_conformance = 'OFF' if args.fast else 'ON'
 
     return (args.cmake_exe, args.build_system_name, args.build_system_call,
-            args.conformance_filter, args.implementation_name,
+            full_conformance, args.conformance_filter, args.implementation_name,
             args.additional_cmake_args, host_names, opencl_names,
             args.additional_ctest_args, args.build_only)
 
-
-def generate_cmake_call(cmake_exe, build_system_name, conformance_filter,
-                        additional_cmake_args, host_names, opencl_names):
+def split_additional_args(additional_args):
+    if additional_args is None:
```

Comment:
Could you add a documentation string here?
I have issues understanding what you are trying to do here.

Reply:
Done. Shlex doesn't support None, so run_conformance_tests.py crashed in case any of "additional arguments" parameter was not set.